### PR TITLE
Update packages page, add AvaloniaUI

### PR DESCRIPTION
**Pull request by worldbeater**
I've tried to make it more obvious that ReactiveUI supports truly cross-platform development using .NET Standard code sharing strategy. Also added info about packages for [AvaloniaUI framework](https://github.com/AvaloniaUI).

### CODE REVIEW

File: input/docs/getting-started/installation/nuget-packages/index.md [hunk 1]
```diff
@@ -15,23 +15,27 @@ Order: 10
...
 
 ```
-- MyCoolApp (application)
-- MyCoolApps.UnitTests (tests)
+- MyCoolApp (netstandard)
+- MyCoolApp.WPF (application)
+- MyCoolApp.UnitTests (tests)
 ```
 
+* Install `ReactiveUI` into your netstandard library, application and tests.
```
Comment (glennawatson) [inline input/docs/getting-started/installation/nuget-packages/index.md]: Winforms/WPF applications tend to have more assemblies due to the cost of assembly loading is much less on those platforms. Maybe reword it to be "Install ReactiveUI into your netstandard libraries, application and tests. I know you are attempting to explain the above project assembly structure but given it's a common use case on these platforms it might be worth explaining it?

Comment (worldbeater) [reply input/docs/getting-started/installation/nuget-packages/index.md]: I agree this makes sence, thanks.

**Comment by glennawatson** [comment]
In terms of those old outdated pages, sounds good to remove them or hide them.
